camelot/view/controls/delegates/one2manydelegate.py

```diff
@@ -77,7 +79,8 @@ def get_standard_item(cls, locale, model_context):
             )
             # dirty hack to keep model context and its name bound as long as
             # the item lives
-            item.setData(one2many_model_context, Qt.ItemDataRole.AccessibleDescriptionRole)
+            #item.setData(one2many_model_context, Qt.ItemDataRole.AccessibleDescriptionRole)
+            one2many_contexts.append(one2many_model_context) # FIXME!!
```

Comment:
ik denk dat dit 'automatisch' gaat/kan w opgelost door gebruik te maken vd CrudItemModel, omdat die de garbage collection gaat triggeren van de model context ?

Reply:
Nee, de QStandardModelItems leven niet lang. Deze worden nu geserialiseerd en niet rechtstreeks gebruikt in het model. De volgende stap zou zijn om die QStandardModelItems in python niet meer aan te maken.